packages/components/typography/src/typography.js

```diff
@@ -0,0 +1,45 @@
+import { TSElement, unsafeCSS, html } from '@tradeshift/elements';
+import css from './typography.css';
+
+customElements.define(
+	'ts-typography',
+	class extends TSElement {
+		constructor() {
+			super();
+			this.text = '';
+			this.color = 'default';
```

Comment:
I saw the color below. It seems like status or type

Reply:
Yeah it can also be a type, I think but I tried to keep it to something like this one: 
https://material-ui.com/api/typography/
Type and status are a little bit vague and because we are only changing colors here, I think it make a little bit more clear for the developers.